lta/lta_cmd.py

```diff
@@ -313,7 +315,11 @@ def date_ok(d: str) -> bool:
     # parse the provided command line arguments and call the function
     args = parser.parse_args()
     if hasattr(args, "func"):
-        await args.func(args)
+        try:
+            await args.func(args)
+        except Exception as e:
+            print(e)
+            asyncio.get_event_loop().stop()
```

Comment:
There seems to be a lot of duplicates of `asyncio.get_event_loop().stop()`.  Here, just below, and in the decorator.  Can these be consolidated into one call?

Or maybe we don't want to `loop.run_forever()` in the first place, but rather `asyncio.run(main())`?